### ISSUE TITLE
Google Chrome: TTS finishes playing current passage some time after pressing pause

### ISSUE THREAD

**Issue by sbwhitt** [internetarchive/bookreader#1246]
**Description**

> What problem are we solving? What does the experience look like today? What are the symptoms? How can one reproduce the problem?

After pausing TTS, the passage will resume after a short amount of time. Then once the passage is finished playback will finally stop. 

**Evidence / Screenshot (if possible)**

> Include evidence of the problem or provide a screenshot.

**Expectation**

> What should be happening? What will it look like / how will it behave when fixed?

**Context**

> What operating system? What browser type and version?

Google Chrome, Windows, no issue on Firefox

**Proposal & Constraints**

> What is the proposed solution / implementation? Is there a precedent of this approach succeeding elsewhere?

I'm assuming this has to do with the [Chrome pausing workaround](https://github.com/internetarchive/bookreader/blob/8adfae834f62b7ea466749dfbc5c04e256173675/src/plugins/tts/WebTTSEngine.js#L328). An additional check could possibly be added to that portion to prevent playback starting up while paused by the user.

**Stakeholders**

> @-tag any stakeholders of this bug

@cdrini

**Comment by ananyakaligal** [comment]
yes @cdrini , here it is 

https://github.com/user-attachments/assets/6f404f0c-e17f-4f0d-80b1-befdf1af7661

**Comment by cdrini** [comment]
Thank you for the video! Ok, so this appears to be an issue with Firefox on Linux not pausing. I cannot reproduce the issue on Windows.

Would you mind creating a new issue for this? I'm going to close this one, since Chrome no longer has the issue initially stated. It was resolved in a recent PR.

Trying to debug to figure out why pause isn't working on Firefox+Linux would be a good one to tackle!
